test/playbooks_reader_test.rb

```diff
@@ -4,90 +4,64 @@
 require_relative '../lib/smart_proxy_ansible/reader_helper'
 
 class PlaybooksReaderTest < Minitest::Test
+  INPUT = JSON.parse(File.read(File.join(__dir__, 'fixtures/playbooks_reader_data.json')))
```

Comment:
could we have this as 
```suggestion
  let(:fixtures) { JSON.parse(File.read(File.join(__dir__, 'fixtures/playbooks_reader_data.json'))) }
```

The global variable seems like not the best practice for a dynamic value based on a file.